lib/platform/gitea/gitea-helper.ts

```diff
@@ -135,7 +136,9 @@ export type RepoSearchParams = {
   archived?: boolean;
 };
 
-export type IssueCreateParams = IssueUpdateParams;
+export type IssueCreateParams = {
+  labels?: number[];
```

Comment:
Why optional here and required below in new interface? Can't we reuse the interface below?

Reply:
It is an optional field in the API when you're creating an issue, but it is required when you use the labels endpoint to update the labels (which is separate from the issue update endpoint).